codegen-core/src/main/kotlin/software/amazon/smithy/rust/codegen/core/smithy/transformers/RemoveEventStreamOperations.kt

```diff
@@ -3,13 +3,13 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-package software.amazon.smithy.rust.codegen.client.smithy.transformers
+package software.amazon.smithy.rust.codegen.core.smithy.transformers
```

Comment:
I'm surprised to see `RemoveEventStreamOperations` used by the server. Was this intentional? It requires allow listing service module names through the `smithy-build.json`.

Reply:
I was surprised too. I've opened https://github.com/awslabs/smithy-rs/pull/1764; once that's merged, I'll keep the transformer in `codegen-client` in this PR.

Reply:
#1764 was merged; I've moved `RemoveEventStreamOperations` back to the client in https://github.com/awslabs/smithy-rs/pull/1762/commits/67dc987505c9aadad62b9e23136071e425a99399.